lib/Remotery.h

```diff
@@ -86,10 +86,10 @@ documented just below this comment.
 // Compiler identification
 #if defined(_MSC_VER)
     #define RMT_COMPILER_MSVC
-#elif defined(__GNUC__)
-    #define RMT_COMPILER_GNUC
 #elif defined(__clang__)
     #define RMT_COMPILER_CLANG
+#elif defined(__GNUC__)
```

Comment:
Depending on the usecase of RMT_COMPILER_* (none so far...), maybe both RMT_COMPILER_GNUC and RMT_COMPILER_CLANG should be defined for clang, as clang supports (almost?) all GCC features?